content/_profiles/content/_articles/OverTheWire-Bandit.md

```diff
@@ -0,0 +1,2525 @@
+
+---
+title: OverTheWIre - Bandit
+author:
+    - Narka
```

Comment:
You might want to make sure this is properly spelled so that it gets linked back to your profile correctly.

Reply:
oh thank you :)